### DISCUSSION TITLE
Ability to Track Grid Usage Charge on Dashboard

### DISCUSSION THREAD

**Discussion by oralallen82** [jasonacox/Powerwall-Dashboard#111] [Ideas]
We currently have savings showing how much we're saving by Solar to home, Solar to grid etc. Do you think we can show usage cost when we do pull from the grid? I have the idea on my head but I'm not sure how to effectively implement it. Currently I am charged $0.134402 per kWh I'd like to see if I pull from the grid due to stormwatch or based on my automation for backup percentage how much that cost me on a daily basis vs how much I am saving.

**Comment by youzer-name** [comment]
@oralallen82 Have you looked at discussion #82 ?

**Reply by oralallen82** [reply]
I did not thanks for pointing this out. I'll give it a gander.
